pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -250,7 +253,7 @@ def __init__(
 
             synchronize_checkpoint_boundary: Insert :func:`torch.cuda.synchronize` at each checkpoint boundary.
 
-            save_full_weights: Gathers weights across all processes before saving to disk
+            load_full_weights: Gathers weights across all processes before saving to disk
```

Comment:
The description is incorrect here, also we need some way to test this (we could simulate this by training without DeepSpeed, and then trying to run predict WITH the deepspeed plugin)